site/docs/skylark/macros.md

```diff
@@ -101,7 +101,7 @@ def chained_genrules(name, visibility=None):
 ```
 
 Note that we only assigned the value of `visibility` to the second genrule.
-This enables macro authors hide outputs of intermediate rules from being
+This enables macro authors to hide the outputs of intermediate rules from being
 depended upon by other targets in the workspace.
```

Comment:
This is a great improvement, thanks!

Since there's now an official [style guide](https://www.bazel.build/basics/docs-style.html), may I offer some more suggestions inspired by it?

- Avoid the words "note" and "we"?
- Use present tense ("assign" vs. "assigned")?

Maybe something like "*This only assigns the value of `visibility` to the second genrule. This lets macro authors hide the outputs...*`

I realize you're not responsible for all that phrasing but this is a good moment to adjust them too.